### ISSUE TITLE
Website tour

### ISSUE THREAD

**Issue by shinuito** [online-go/online-go.com#1709]
**Is your feature request related to a problem? Please describe.**

It seems many people, many many people find OGS very unintuitive. Whether it’s the layout, the home page, the hamburger menu, the wording on the challenges etc.

E.g see https://forums.online-go.com/t/how-intuitive-is-ogs-for-new-users/31614?u=shinuito

and most recently again https://forums.online-go.com/t/how-to-search-for-a-player/41483/3

While some might have ideas for a layout overhaul 

https://forums.online-go.com/t/how-intuitive-is-ogs-for-new-users/31614/164?u=shinuito

and replies, or even redesign of the home page, it would probably take a lot of work.

**Describe the solution you'd like**
What about a short terms solution of a site tour button, put it on the homepage, or any other page that might need explanation. Would it be possible to make a react component out of it, and pass in some page specifics, like the page elements/components to it?

**Additional context**
Basically I can imagine either pop up boxes or highlights elements with css, either borders or some sort of shadow to make a glow effect along with some text.

Then one can maybe add an end tour/hide all tours button, and possibly something in settings to reset the tours.

**Comment by benjaminpjones** [comment]
Here's a couple of libraries that might help (or at least provide inspiration):

https://react-joyride.com/
https://shipshapecode.github.io/react-shepherd/

**Comment by shinuito** [comment]
Even a basic button that just highlights the hamburger menu and says “this is a menu, it can be clicked” might be a huge improvement :)

Could take it step by step. The links look interesting

**Comment by anoek** [comment]
I'm a bit torn, on the one hand I agree that things are complex and confusing enough that a tour might benefit some folks. On the other hand, I would love to simplify and redesign a bit so that it's not so confusing, so that a tour wouldn't be necessary.

**Comment by Fraze** [comment]
> Even a basic button that just highlights the hamburger menu and says “this is a menu, it can be clicked” might be a huge improvement :)

Another solution would be to start with the menu open on the home page (and also have it push the page, not hover over). Then people can use hamburger to declutter their screen if they so choose, but otherwise the items are immediately available.

**Comment by jaysephjw** [comment]
@shinuito I just [posted](https://forums.online-go.com/t/how-intuitive-is-ogs-for-new-users/31614/239) to one of those threads,  thanks for including! Posted re addressing UX changes.

- 

For a simple onboarding experience though, I might offer directing people to the wiki? The 'getting started' page is pretty nice, but the wiki is a little buried / unfriendly to find. 

I do generally agree w anoek philisophically that the site should speak for itself on usability, but a stop-gap in the interim could be good too.
